scripts/itest_part.sh

```diff
@@ -11,6 +11,15 @@ NUM_TRANCHES=$2
 shift
 shift
 
+# Start postgres
+if [[ $DBBACKEND == "postgres" ]]; then
+  docker rm -f lnd-postgres
+  docker run --name lnd-postgres -e POSTGRES_PASSWORD=postgres -p 6432:5432 -d postgres
```

Comment:
Echoing offline comment from @bhandras here:

>for itests you just need to make sure to use ports generated by the port generator helper so that when we run them in parallel they don't clash

Another option could be to run a single db instance at a higher level that all tranches use?

Reply:
Currently itests can run in parallel sparing us a _lot_ of CI time so I think it's important to support that.

Reply:
Yes, I mean keep running the itests in parallel, but let them all connect to the same postgres instance.

Reply:
Updated the itest commit to show how it can be done.